services/management/dbaas/components_service.go

```diff
@@ -34,6 +37,13 @@ type componentsService struct {
 	versionServiceClient versionService
 }
 
+type disablingModifier int
+
+const (
+	equal disablingModifier = iota
```

Comment:
We don't need this enum, we can just remove everything less then 8.0.0 for PXC

Reply:
removed